routing/payment_lifecycle.go

```diff
@@ -341,8 +341,8 @@ func (p *paymentLifecycle) sendPaymentAttempt(firstHop lnwire.ShortChannelID,
 		return err
 	}
 
-	log.Debugf("Payment %x (pid=%v) successfully sent to switch",
-		p.payment.PaymentHash, p.attempt.PaymentID)
+	log.Debugf("Payment %x (pid=%v) successfully sent to switch, route: %v",
```

Comment:
hm, won't this be quite spammy? Maybe it is okay, since it is useful information indeed.

Reply:
It is still a single log line

Reply:
Yeah true, just  a potentially long line 😂